bot/utils/roleParsing.js

```diff
@@ -24,7 +24,7 @@ function splitHeader(text) {
 function parseRoleMessage(text) {
     const parts = splitHeader(text);
 
-    const roleRegex = /^\s*((?:<:)?[^:]*(?::[0-9]+>)?)\s+:\s+([^:]*)(?:\s+:\s+(.*))?$/;
+    const roleRegex = /^\s*((?:<:)?[^:]*(?::[0-9]+>)?)\s*:\s*([^:]*)(?:\s*:\s*(.*))?$/;
```

Comment:
We should prob have some tests for this regex

Reply:
There is a test that checks that the `parseRoleMessage()` function behaves correctly but not one specifically for this regex. It would seem when I just tried to add the regex safety check to this it fails so this will need to be reworked. It is too large of a scope for this PR and a separate issue will be opened  for it.